.editorconfig

```diff
@@ -0,0 +1,30 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = lf
+max_line_length = 127
```

Comment:
this contradicts google style it is regulated for java to 100 characters

Reply:
So you wanna keep 127 as default value and force 100 for java ?

Reply:
Since we use googleJavaFormat("1.25.2").aosp().reorderImports(false), the editor should also know this https://google.github.io/styleguide/javaguide.html